### ISSUE TITLE
Add caption and alt to the images

### ISSUE THREAD

**Issue by ipeychev** [liferay/alloy-editor#118]

**Comment by BenjaminBini** [comment]
It would be a greate feature.

**Comment by chipcullen** [comment]
The lack of alt text is an accessibility issue. Personally, I think alt text should be prioritized above  caption text.

**Comment by jbalsas** [comment]
We actually implemented `alt` separately inside `Liferay Portal`. It should be pretty straightforward to bring it here and bundle it by default...

https://github.com/liferay/liferay-portal/blob/65059440dfaf2b8b365a20f99e83e4cdb15478aa/modules/apps/frontend-editor/frontend-editor-alloyeditor-web/src/main/resources/META-INF/resources/js/extras/buttons/accessibility/accessibility_image_alt.jsx

**Comment by wincent** [comment]
Doing some "spring cleaning" by closing issues that were opened in 2015 — much has changed since then and there have been many releases, so there is a risk that these older issues have grown very stale and are very unlikely to be acted upon. Please feel free to commend and we can look at re-opening on a case-by-case basis.

**Comment by JorgeFerrer** [comment]
Did we ever implement the ability to add alt and/or caption to images to make it possible for content authors to comply with accessibility?

**Comment by JorgeFerrer** [comment]
I just saw Chema's comment above that it was implemented within Liferay Portal. That's good enough for me.

**Comment by julien** [comment]
@JorgeFerrer actually since 2.x this is now part of AlloyEditor: one of the goals of the 2.x release was to include everything that was in portal in the alloy-editor repository.

**Comment by JorgeFerrer** [comment]
Awesome, thanks Julien

**Comment by d80wrk** [comment]
Caption has never been included as a feature. Bizarre.
